README.md

```diff
@@ -19,8 +19,8 @@ tracking.
 
 ### Pre-requisites
 Following are the prerequisites to build JDK on Mac targeting iOS:
-1. Download and unzip pre-built JDK24 to be used as boot JDK
-2. Download the [support zip](https://download2.gluonhq.com/mobile/mobile-support-20250106.zip) which contains an ios build for libffi and cups. Unzip it in the root directory of the project.
+1. Download and install JDK24 for macOS. You can use the JDK you just built targeting macOS in the above note instead, but this is not recommended.
```

Comment:
out of curiosity, why is that not recommended?

Reply:
I was conservative and thought we wouldn't want to encourage using a bleeding edge JDK compiled off master as the boot JDK. If this is ok, I can remove that "Not recommended" note.